.github/workflows/format.yml

```diff
@@ -0,0 +1,77 @@
+name: Format
```

Comment:
Typically such tests are more often called as lint/linter checks; it's be a better name considering that Format is a stdlib module and so for an OCaml project, it's a bit ambiguous to see that as a standalone name.